index.html

```diff
@@ -0,0 +1,312 @@
+              class="person-card__social-icon">
+          </div>
+        </div>
+      </div>
+      <div class="person-card">
+        <img src="./assets/images/cover.png" alt="" class="person-card__image">
+        <div class="person-card__details">
+          <h5 class="headline headline--dark headline--bolder headline--big">Ashley Fletcher</h5>
+          <h6 class="headline">Consultant</h6>
+          <div class="person-card__socials-container">
+            <img src="./assets/icons/ant-design_facebook-filled.svg" alt="facebook icon"
+              class="person-card__social-icon">
+            <img src="./assets/icons/ant-design_instagram-outlined.svg" alt="instagram icon"
+              class="person-card__social-icon">
+            <img src="./assets/icons/ant-design_twitter-outlined.svg" alt="twitter icon"
+              class="person-card__social-icon">
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="contact-us">
+    <div class="section-text section-text--left-aligned">
+      <p class="description">Contact us</p>
+      <h1 class="section-text__heading">Meet Our Team</h1>
+    </div>
+    <form action="" class="contact-us__form">
+      <input class="input" placeholder="Full name" type="text" name="" id="">
+      <input class="input" placeholder="Full name" type="text" name="" id="">
+      <input class="input" placeholder="Full name" type="text" name="" id="">
+      <input class="input" placeholder="Full name" type="text" name="" id="">
+      <textarea class="input input--big" placeholder="Message" name="" id="" cols="30" rows="10"></textarea>
+      <button type="submit" class="button button--primary">Book appointment</button>
+    </form>
+  </section>
+  <section class="consulting-section">
+    <div class="section-text section-text--left-aligned">
+      <h1 class="section-text__heading">Consulting Agency For Your Business</h1>
+      <p class="description">the quick fox jumps over the lazy dog</p>
+    </div>
+    <button type="button" class="button button--primary">Contact Us</button>
+  </section>
+  <footer class="footer">
+    <div class="footer__upper-section">
+      <h5 class="logo logo--white">Kmarketer</h5>
+      <div class="footer__socials-container">
+        <img class="footer__social-icon" src="./assets/icons/ant-design_facebook-filled (1).svg" alt="facebook icon">
+        <img class="footer__social-icon" src="./assets/icons/ant-design_instagram-outlined (1).svg"
+          alt="Instagram icon">
+        <img class="footer__social-icon" src="./assets/icons/ant-design_twitter-outlined (1).svg" alt="Twitter icon">
+        <img class="footer__social-icon" src="./assets/icons/carbon_logo-youtube.svg" alt="youtube icon">
+      </div>
+    </div>
+    <hr class="footer__separator">
+    <div class="footer__lower-section">
+      <ul class="footer__links">
+        <li class="headline headline--big headline--bolder headline--white">
```

Comment:
Again here if you name it `headline--white`, it is not a good idea because if that changes in the future, you would have to change it in HTML and CSS. Something like `headline--primary` or `headline--light` is a good idea.

Reply:
Yeah sure!
thanks